packages/react-error-overlay/src/index.js

```diff
@@ -167,3 +167,24 @@ window.__REACT_ERROR_OVERLAY_GLOBAL_HOOK__.iframeReady = function iframeReady()
   isLoadingIframe = false;
   updateIframeContent();
 };
+
+const warningText =
+  'When deploying an application, `react-error-overlay` should be excluded ' +
+  'as it is a heavy dependency meant for development.\n\n' +
+  'Consider adding an error boundary to your tree to customize error ' +
```

Comment:
Why do we need the error boundary text here? Doesn't seem very relevant.

Reply:
If someone was purposely including `react-error-overlay` for production errors, though I know most cases would be accidental.

We can remove this.